noseparallel/plugin.py

```diff
@@ -25,7 +28,11 @@ def wantMethod(self, method):
         return None
 
     def _pick_by_hash(self, name):
```

Comment:
This no longer uses name and is not a "pick by hash", you should remove the argument and change the name.